lib/local_system.rb

```diff
@@ -67,6 +67,16 @@ def validate_existence_of_packages(packages)
       end
     end
 
+    def validate_existence_of_command(command, package)
+      Cheetah.run("which", command)
+    rescue Cheetah::ExecutionFailed
+      output = <<-EOF
+You need the command '#{command}' from package '#{package}'.
+You can install it by running `zypper install #{package}`.
```

Comment:
We shouldn't recommend using zypper if this check is mainly targeted for other distributions.